subsystems/drivebase/SwerveDrive.java

```diff
@@ -225,8 +228,13 @@ public void mustangPeriodic() {
             }
         }
         mPoseEstimator.update();
-        SmartDashboard.putNumber("navX heading", getPose().getRotation().getDegrees());
-        SmartDashboard.putNumber("pitch", getPitch());
+        Logger.getInstance().recordOutput(getName()+"/navX Heading Deg", getPose().getRotation().getDegrees());
+        Logger.getInstance().recordOutput(getName()+"/pitch",getPitch());
+        Logger.getInstance().recordOutput(getName()+"/roll",getRoll());
```

Comment:
why do we call Logger.getInstance() repeatedly rather than just once and reusing the variable? Is there a need to call it once per periodic even? Could we stash it in the object instance? Or even as a class variable that we use everywhere effectively as a global? Depending how it works, one or more of those options might not be possible (if it's doing something custom per invocation or per object) but at minimum I expect we can reuse it within this method.

Related: is there a version of recordOutput() that takes multiple key/value pairs to avoid making three calls?

Is there a return value that we should be looking at? Or even if there is, do we not care?

Reply:
There is no return value and it doesn't like there are any methods for pushing multiple key-value pairs  in one call. Also, Logger has become static
https://github.com/Mechanical-Advantage/AdvantageKit/blob/main/junction/core/src/org/littletonrobotics/junction/Logger.java